### PR TITLE
Makes Ghost HUD toggle persistent

### PR DESCRIPTION
**Pull request by AnturK**
:cl: AnturK
tweak: Ghost HUD and Ghost Inquisitiveness  toggles are now persistent between rounds.
/:cl:

### CODE REVIEW

**Comment by RemieRichards** [comment]
@AnturK while you're touching ghosts and save files can you PLEASE make inquisitiveness a pref? it annoys me to hell and back having to turn it off every round.

**Comment by AnturK** [comment]
Sure

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -181,6 +181,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["ghost_orbit"]		>> ghost_orbit
 	S["preferred_map"]		>> preferred_map
 	S["ignoring"]			>> ignoring
+	S["ghost_hud"]			>> ghost_hud
+	S["inquisitive_ghost"]			>> inquisitive_ghost
```
Comment (RemieRichards) [inline code/modules/client/preferences_savefile.dm]: the indentation here is fubar

File: code/modules/client/preferences_savefile.dm [hunk 2]
```diff
@@ -220,6 +222,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["ghost_orbit"]		<< ghost_orbit
 	S["preferred_map"]		<< preferred_map
 	S["ignoring"]			<< ignoring
+	S["ghost_hud"]			<< ghost_hud
+	S["inquisitive_ghost"]	<< inquisitive_ghost
```
Comment (RemieRichards) [inline code/modules/client/preferences_savefile.dm]: >the indentation here is fubar

Comment (AnturK) [inline code/modules/client/preferences_savefile.dm]: That's because github uses size 8 spaces, ew. ?ts=4 in url to change size

**Comment by LatD** [comment]
I'd like other ghost toggle things like this too, especially toggle darkness and ghost health/sec/diag HUD since I turn them always on

**Comment by KazeEspada** [comment]
Yes, toggle darkness.

**Comment by Anonus** [comment]
If toggle darkness/ghost vision get prefs, can the button not toggle the pref? I like to toggle both midround but don't want to accidentally default to a setting I don't want to default to.

File: code/modules/client/preferences_toggles.dm [hunk 3]
```diff
@@ -253,3 +253,25 @@ var/global/list/ghost_orbits = list(GHOST_ORBIT_CIRCLE,GHOST_ORBIT_TRIANGLE,GHOS
...
+/client/verb/toggle_inquisition() // warning: unexpected inquisition
+	set name = "Toggle Inquisitiveness"
+	set desc = "Sets whether your ghost examines everything on click by default"
+	set category = "Preferences"
+
+	prefs.inquisitive_ghost = !prefs.inquisitive_ghost
+	if(prefs.inquisitive_ghost)
+		src << "<span class='notice'>You will now examine everything you click on.</span>"
+	else
+		src << "<span class='notice'>You will no longer examine things you click on.</span>"
```
Comment (tkdrg) [inline code/modules/client/preferences_toggles.dm]: Don't you need a `prefs.save_preferences()` here?

**Pull request merged**